clickup/custom_fields.go

```diff
@@ -339,7 +339,7 @@ func getDateValue(v interface{}) (time.Time, bool) {
 		return time.Time{}, false
 	}
 
-	return time.Unix(0, num*int64(time.Millisecond)), true
+	return time.UnixMilli(num), true
```

Comment:
`time.UnixMilli()` is available in go1.17 or later. It's important for ClickUp API Client.